torchdynamo/variables/lists.py

```diff
@@ -305,6 +305,16 @@ def __init__(self, items, **kwargs):
             start, stop, step = items
         else:
             assert False
+
+        # Avoids a .item() call in the tensor slice that would attempt to get a
+        # value out fake tensors, and which would determine the output shape of
+        # the slice.  It is a workaround until
+        # https://github.com/pytorch/pytorch/pull/83567 is landed and there is
+        # more complete support for breaking on data dependent operators.
+        for limit in (start, stop, step):
```

Comment:
We have a flag for this, in config, capture_scalar_outputs - do you want to check if this is enabled? It would help us avoid this graph break in export.